specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-02-02-preview/managedClusters.json

```diff
@@ -3984,6 +3984,10 @@
           },
           "title": "PublicNetworkAccess of the managedCluster",
           "description": "Allow or deny public network access for AKS"
+        },
+        "kmsProfile": {
+          "$ref": "#/definitions/ManagedClusterKMSProfile",
+          "description": "The Key Management Service configuration."
```

Comment:
Is there a link to Key Management Service documentation? Why might I want to set this? What capabilities does it enable? See other documentation in this file and consider including a link here like some of them do.

Reply:
fixed.